hybridse/src/planv2/planner_v2_test.cc

```diff
@@ -331,50 +331,45 @@ TEST_F(PlannerV2Test, MultiProjectListPlanPostTest) {
     node::PlanNodeList trees;
     base::Status status;
     const std::string sql =
-        "SELECT sum(col1) OVER w1 as w1_col1_sum, "
-        "sum(col3) OVER w2 as w2_col3_sum, "
-        "sum(col4) OVER w2 as w2_col4_sum, "
-        "col1, "
-        "sum(col3) OVER w1 as w1_col3_sum, "
-        "col2, "
-        "sum(col1) OVER w2 as w2_col1_sum, "
-        "lag(col1, 0) OVER w2 as w2_col1_at_0, "
-        "lag(col1, 1) OVER w2 as w2_col1_at_1 "
-        "FROM t1 "
-        "WINDOW "
-        "w1 AS (PARTITION BY col2 ORDER BY `TS` ROWS_RANGE BETWEEN 1d "
-        "PRECEDING AND "
-        "1s PRECEDING), "
-        "w2 AS (PARTITION BY col3 ORDER BY `TS` ROWS_RANGE BETWEEN 2d "
-        "PRECEDING AND "
-        "1s PRECEDING) "
-        "limit 10;";
-    std::cout << sql << std::endl;
+        R"sql(SELECT
+        sum(col1) OVER w1 as w1_col1_sum,
+        sum(col3) OVER w2 as w2_col3_sum,
+        sum(col4) OVER w2 as w2_col4_sum,
+        col1,
+        sum(col3) OVER w1 as w1_col3_sum,
+        col2,
+        sum(col1) OVER w2 as w2_col1_sum,
+        lag(col1, 0) OVER w2 as w2_col1_at_0,
+        lag(col1, 1) OVER w2 as w2_col1_at_1
```

Comment:
maybe we can move one `lag` to the middle of the projection list to check the ordering is correct

Reply:
added